startup/40-xspress3.py

```diff
@@ -262,7 +264,7 @@ def set_channels_for_hdf5(self, channels=(1, 2, 3, 4, 5, 6)):
     # Currently only using four channels. Uncomment these to enable more
     # channels:
     # channel5 = C(Xspress3Channel, 'C5_', channel_num=5)
-    channel6 = C(Xspress3Channel, 'C6_', channel_num=6)
+    # channel6 = Cpt(Xspress3Channel, 'C6_', channel_num=6)
     # channel7 = C(Xspress3Channel, 'C7_', channel_num=7)
     # channel8 = C(Xspress3Channel, 'C8_', channel_num=8)
```

Comment:
I think those `C` in comments should be also changed to `Cpt` to avoid failures if uncommented.